web/sw-recv.js

```diff
@@ -0,0 +1,43 @@
+
+var _cacheName = 'cimbar-recv-js-vVERSION';
+var _cacheFiles = [
```

Comment:
I'm still concerned whether I'm doing this service worker caching stuff right. We'll see :grimacing: